### ISSUE TITLE
Different lores for genshop vs item purchased

### ISSUE THREAD

**Issue by syremc** [arcadiadevs/gensplus-issues#52]
### **Is your feature request related to a problem? Please describe.**

Currently when purchasing a generator, the lore is the exact same for when the the generator is purchased by a user and within their inventory they can hover the generator and see a lore with information on it.

### **Describe the solution you'd like**

I want to add different lores for the genshop so when users are buying a generator, they see at the bottom of the gen lore, 'Click to buy this item' but when they finally purchase said generator, they will no longer see 'Click to buy this item'.

Basically separate the lores for the shop & the items itself.

### **Additional context**
![image11](https://github.com/arcadiadevs/gensplus-issues/assets/107783232/f037158c-1dce-4cb1-96f1-5d062c74b824)
![image111](https://github.com/arcadiadevs/gensplus-issues/assets/107783232/43423cb8-54ae-43f4-b6ad-d8cd11c0cf07)

The lores are the same in these photos, but I want to be able to make these different.
